### PR TITLE
Devsite requirements: Bump Django point versions to most recent

### PR DESCRIPTION
**Pull request by johnbaldwin**

### CODE REVIEW

File: devsite/requirements/hawthorn_community.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ pytz==2016.10
 ## Django package dependencies
 ##
 
-Django==1.11.15
+Django==1.11.28
```
Comment (OmarIthawi) [inline devsite/requirements/hawthorn_community.txt]: it's `Django==1.11.23` in hawthorn

```suggestion
Django==1.11.23
```

File: devsite/requirements/hawthorn_multisite.txt [hunk 2]
```diff
@@ -23,7 +23,7 @@ pytz==2016.10
 ## Django package dependencies
 ##
 
-Django==1.11.15
+Django==1.11.28
```
Comment (OmarIthawi) [inline devsite/requirements/hawthorn_multisite.txt]:
```suggestion
Django==1.11.23
```

File: devsite/requirements/ginkgo.txt [hunk 3]
```diff
@@ -25,7 +25,7 @@ pytz==2016.7
 ## Django package dependencies
 ##
 
-Django==1.8.18
+Django==1.8.19
```
Comment (OmarIthawi) [inline devsite/requirements/ginkgo.txt]: It's `Django==1.8.18` in `open-release/ginkgo.master`. Not sure about our fork.

```suggestion
Django==1.8.18
```

**Comment by johnbaldwin** [comment]
@OmarIthawi I set the versions to those you specified to match the version used in the platform

**Pull request merged**